formula-android/src/main/java/com/instacart/formula/integration/ActivityStoreContext.kt

```diff
@@ -56,6 +48,21 @@ abstract class ActivityStoreContext<out Activity : FragmentActivity> {
      */
     abstract fun isFragmentResumed(contract: FragmentContract<*>): Observable<Boolean>
 
+    /**
+     * This enables you to select specific events from the activity.
+     *
+     * @param Event Type of event
+     */
+    abstract fun <Event> selectActivityEvents(
+        select: Activity.() -> Observable<Event>
+    ): Observable<Event>
+
+    /**
+     * Performs an [effect] on the current activity instance. If there is no activity connected,
```

Comment:
oh actually it is documented at least, so not that bad .. as long as people read the doc 😬